SwiftUIDZ/SwiftUIDZ/CarsView.swift

```diff
@@ -0,0 +1,246 @@
+                            .foregroundColor(.white)
+                            .padding()
+                            .offset(x: CGFloat(offset))
+                    }
+                    Image(cars[segmentIndex].name)
+                        .resizable()
+                        .scaledToFit()
+                        .offset(x: CGFloat(offset))
+                }
+                .animation(.spring(response: 0.3, dampingFraction: 0.7, blendDuration: 1))
+                HStack(spacing: Constants.smallSpacing) {
+                    createCarParameter(
+                        imageName: Constants.meterImageName,
+                        parameterText: cars[segmentIndex].speed
+                    )
+                    createCarParameter(
+                        imageName: Constants.seatImageName,
+                        parameterText: cars[segmentIndex].seats
+                    )
+                    createCarParameter(
+                        imageName: Constants.engineImageName,
+                        parameterText: cars[segmentIndex].power
+                    )
+                }
+                .padding()
+                .frame(height: Constants.commonSize)
+                .foregroundColor(Color(UIColor.systemGray6))
+                createCarNameSegment()
+                createForm()
+                Text(Constants.distanceText)
+                    .font(Font.custom(Constants.fontName, size: Constants.fontSizeRegular).weight(.bold))
+                    .foregroundColor(.white)
+                createDistanceSlider()
+                HStack(spacing: Constants.spacing) {
+                    Text(toggleIsOn ? "\(Constants.totalText)" +
+                          " \(rents[segmentIndexRentalPeriod].price + Constants.oneHundred) $" :
+                            " \(Constants.totalText) \(rents[segmentIndexRentalPeriod].price) $")
+                    .font(Font.custom(Constants.fontName, size: Constants.fontSizeRegular).weight(.bold))
+                    createRentButton()
+                    createShareButton()
+                }
+                .padding()
+                .navigationBarHidden(true)
+            }
+        }
+    }
+    
+    // MARK: - Private Methods
+    
+    private func createCarParameter(imageName: String, parameterText: String) -> some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: Constants.cornerRadius)
+            VStack {
+                Image(imageName)
+                    .resizable()
+                    .renderingMode(.template)
+                    .foregroundColor(.purple)
+                    .frame(width: Constants.parameterIconSize, height: Constants.parameterIconSize)
+                Text(parameterText)
+                    .foregroundColor(.white)
+                    .font(Font.custom(Constants.fontName, size: Constants.fontSizeSmall).weight(.bold))
+            }
+        }
+    }
+    
+    private func createCarNameSegment() -> some View {
```

Comment:
```suggestion
    private var carNameSegmentView: some View {
```
Нет параметров делай вычисляемым и ниже так же